src/content/docs/dropins/all/extend-or-create.mdx

```diff
@@ -0,0 +1,193 @@
+---
+title: Extend, substitute, or create?
+description: Decision guidance for customizing drop-ins - when to extend existing ones, when to substitute with third-party solutions, and when creating new ones might be necessary.
+sidebar:
+  label: Extend, substitute, or create
+---
+
+import Vocabulary from '@components/Vocabulary.astro';
+import Diagram from '@components/Diagram.astro';
+import Aside from '@components/Aside.astro';
+import Callouts from '@components/Callouts.astro';
+import { Steps } from '@astrojs/starlight/components';
+import Tasks from '@components/Tasks.astro';
+import Task from '@components/Task.astro';
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+When customizing Commerce drop-ins, you have three main approaches. **Extending** existing drop-ins is the recommended and fully supported path. **Substituting** drop-ins with third-party solutions is sometimes necessary but comes with risks. **Creating** new drop-ins from scratch is technically possible but currently experimental and unsupported. This guide helps you choose the right approach for your specific needs.
+
+## Vocabulary
+
+<Vocabulary>
+
+### Extend
+
+Modify or enhance existing drop-ins using slots, styling, events, or configuration. This is the **recommended and fully supported** approach.
+
+### Substitute
+
+Replace an Adobe drop-in entirely with a third-party solution (like using Stripe for checkout). This comes with maintenance responsibilities and compatibility risks.
+
+### Create
+
+Build a brand-new drop-in from scratch using the SDK. Currently in **early access** - experimental with limited support.
+
+### Slots
+
+Declarative anchors where custom UI content can be injected into a drop-in.
+
+</Vocabulary>
+
+## Decision guidance
+
+Use this decision flow to determine the right approach for your needs:
+
+<Steps>
+1. **Can you achieve your goals by extending?** 
+   - Use slots to add custom content
+   - Apply custom styling and layouts
+   - Listen to events for custom behavior
+   - Modify configurations
+   
+   **If yes** → **Extend** (recommended path)
+
+2. **Do you need to completely replace core functionality?**
+   - Common with checkout flows (Stripe, PayPal, etc.)
+   - Third-party service provides their own UI/logic
+   - Adobe's drop-in doesn't fit your business model
+   
+   **If yes** → **Substitute** (proceed with caution)
+
+3. **Do you need something entirely new that doesn't exist?**
+   - Building for reuse across multiple storefronts/brands
+   - No existing drop-in covers your use case
+   - You understand the maintenance responsibilities
+   
+   **If yes** → **Create** (early access, high risk)
```

Comment:
"high risk" might be too strongly worded. early access might be enough.

Reply:
I came to say the same.